monolith/lib/parseQoh.ts

```diff
@@ -1,4 +1,4 @@
-interface QOH {
+export interface QOH {
```

Comment:
We should pull this and `ScryfallCard` into its own `./returnTypes` directory in `./interactors` to centralize all the possible db return types